DataFusion/src/V1/Gapic/DataFusionGapicClient.php

```diff
@@ -307,19 +437,16 @@ public function __construct(array $options = [])
      *
      * @throws ApiException if the remote call fails
      */
-    public function createInstance(array $optionalArgs = [])
```

Comment:
This breaks BC, but that is noted in the proto update, and as this library is in BETA, this is acceptable.